### PR TITLE
docs: Update README.md

### PR DESCRIPTION
**Pull request by sthirthala**
Added export specification

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,12 @@ Generate API SDK clients for your favorite language including JavaScript, TypeSc
 
 ![generate-api-client](./media/generate-api-client.png)
 
+### Export API specification
+
+Export API specification document for a specific API either from the tree view or from the command palette
+
+![Export API specification](./media/export-api.gif)
+
 ## Notice/Known Issues
```
Comment (formulahendry) [inline README.md:49]: Remove 'Notice/Known Issues' section

**Review by formulahendry — CHANGES_REQUESTED** [review]
Also remove section and gif of GitHub Copilot Chat

File: README.md [hunk 2]
```diff
@@ -40,11 +37,12 @@ Generate API SDK clients for your favorite language including JavaScript, TypeSc
 
 ![generate-api-client](./media/generate-api-client.png)
 
-## Notice/Known Issues
+### Export API specification
```
Comment (formulahendry) [inline README.md:40]: I think it is better to move this into the 'Explore' section, and no need for `## Export API specification`

Comment (sthirthala) [reply README.md]: I merged it with Consume section. Please check. It better fits into Consume section as the portal also works that way.

**Review by wenytang-ms — APPROVED** [review]

**Review by formulahendry — APPROVED** [review]

**Pull request merged**